CHANGELOG.md

```diff
@@ -0,0 +1,55 @@
+# Changelog
+
+## 0.0.0a8
+
+### Changes
+
+- Add --config CLI arg ([#17697](https://github.com/astral-sh/ruff/pull/17697))
+- Add CLI documentation and update README ([#284](https://github.com/astral-sh/ty/pull/284))
+- Add a warning about pre-release status to the CLI ([#17983](https://github.com/astral-sh/ruff/pull/17983))
+- Add missing bitwise-operator branches for boolean and integer arithmetic ([#17949](https://github.com/astral-sh/ruff/pull/17949))
+- Add progress bar for `ty check` ([#17965](https://github.com/astral-sh/ruff/pull/17965))
+- CLI reference ([#17978](https://github.com/astral-sh/ruff/pull/17978))
+- Change default severity for `unbound-reference` to `error` ([#17936](https://github.com/astral-sh/ruff/pull/17936))
+- Change range of `revealed-type` diagnostic to be the range of the argument passed in, not the whole call ([#17980](https://github.com/astral-sh/ruff/pull/17980))
+- Default to latest supported python version ([#17938](https://github.com/astral-sh/ruff/pull/17938))
+- Display "All checks passed!" message in green ([#17982](https://github.com/astral-sh/ruff/pull/17982))
+- Document configuration schema ([#17950](https://github.com/astral-sh/ruff/pull/17950))
+- Generate and add rules table ([#17953](https://github.com/astral-sh/ruff/pull/17953))
+- Handle typevars that have other typevars as a default ([#17956](https://github.com/astral-sh/ruff/pull/17956))
+- Ignore `possibly-unresolved-reference` by default ([#17934](https://github.com/astral-sh/ruff/pull/17934))
+- Implement `global` handling and `load-before-global-declaration` syntax error ([#17637](https://github.com/astral-sh/ruff/pull/17637))
+- Make `unused-ignore-comment` disabled by default for now ([#17955](https://github.com/astral-sh/ruff/pull/17955))
+- Recognise functions containing `yield from` expressions as being generator functions ([#17930](https://github.com/astral-sh/ruff/pull/17930))
+- Recursive protocols ([#17929](https://github.com/astral-sh/ruff/pull/17929))
+- Remove `SliceLiteral` type variant ([#17958](https://github.com/astral-sh/ruff/pull/17958))
+- Report duplicate `Protocol` or `Generic` base classes with `[duplicate-base]`, not `[inconsistent-mro]` ([#17971](https://github.com/astral-sh/ruff/pull/17971))
+- Respect the gradual guarantee when reporting errors in resolving MROs ([#17962](https://github.com/astral-sh/ruff/pull/17962))
+- Support `typing.Self` in methods ([#17689](https://github.com/astral-sh/ruff/pull/17689))
+- Support extending `__all__` from an imported module even when the module is not an `ExprName` node ([#17947](https://github.com/astral-sh/ruff/pull/17947))
+- Support extending `__all__` with a literal tuple or set as well as a literal list ([#17948](https://github.com/astral-sh/ruff/pull/17948))
+- Understand classes that inherit from subscripted `Protocol[]` as generic ([#17832](https://github.com/astral-sh/ruff/pull/17832))
+- Update salsa ([#17937](https://github.com/astral-sh/ruff/pull/17937))
```

Comment:
```suggestion
```

Reply:
Yeah I guess I was using a too-broad definition of "interesting to users" -- knowing Salsa was updated could be relevant to a user trying to understand some unintended behavior change, but at that point it's a "contributor" not a "user" and they can look at the github history, too